server/migrations/1696004922504_add-not-null-pool-spotify-id.sql

```diff
@@ -0,0 +1,9 @@
+-- Up Migration: Pools now must have a spotify_id and if we delete a pool, we delete the slot related to it as well
+ALTER TABLE slot DROP CONSTRAINT slot_pool_id_fkey;
+ALTER TABLE slot ADD CONSTRAINT slot_pool_id_fkey FOREIGN KEY (pool_id) REFERENCES pool(id) ON DELETE CASCADE;
+DELETE FROM pool WHERE spotify_id IS NULL;
```

Comment:
realized afterwards I probably shouldn't have a delete in a migration and should have done it as two different migrations, but it would probably be a mess to try and fix the migrations now